Cargo.toml

```diff
@@ -28,25 +30,27 @@ gloo-net = { version = "0.4", features = ["http"] }
 log = "0.4"
 cfg-if = "1.0"
 tracing = "0.1"
-reqwest = { version = "0.11", features = ["json"] }
 getrandom = { version = "0.2", features = ["js"] }
-lemmy_api_common = { tag = "0.19.0-rc.1", git = "https://github.com/LemmyNet/lemmy.git", default-features = false }
+lemmy_api_common = { git = "https://github.com/LemmyNet/lemmy.git", branch = "main", default-features = false }
 
+wasm-cookies = "0.2.1"
```

Comment:
i see you made this more specific than before, this would have been picked up by a cargo update of version "0.2"

is there a reason for being more specific?

Reply:
No. I can put it back to 0.2.

Reply:
Also be sure to have `cargo edit` installed on your systems, and use `cargo upgrade`, which will overwrite a lot of these anyway.